### PR TITLE
feat [#W1.0] end to end tests

### PR DESCRIPTION
**Pull request by Stefann11**

### CODE REVIEW

**Review by trki293 — APPROVED** [review]
PR is approved. Good Clean code principles are followed.

File: PatientWebApplication/PatientWebAppSeleniumTests/CreateFeedbackTests.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    public class CreateFeedbackTests : IDisposable
+    {
+        private readonly IWebDriver driver;
+        private Pages.FeedbackPage feedbackPage;
+        private Pages.CreateFeedbackPage createFeedbackPage;
+        private int feedbackCount = 0;
+
+        public CreateFeedbackTests()
+        {
+            // options for launching Google Chrome
```
Comment (velicko-tam) [inline PatientWebApplication/PatientWebAppSeleniumTests/CreateFeedbackTests.cs:19]: Remove unnecessary comments

**Review by velicko-tam — CHANGES_REQUESTED** [review]
PR is not approved. Marked parts should be changed.

**Review by velicko-tam — APPROVED** [review]
PR is approved.

**Pull request merged**